src/main/resources/assets/admin/common/js/content/Widget.ts

```diff
@@ -43,6 +44,14 @@ export class Widget<B extends WidgetBuilder = WidgetBuilder, C extends WidgetCon
         return this.iconUrl;
     }
 
+    public getFullUrl(): string {
+        return `${CONFIG.getString('widgetApiUrl')}/${this.url}`;
+    }
+
+    public getFullIconUrl(): string {
+        return `${CONFIG.getString('widgetApiUrl')}${this.iconUrl}`;
+    }
+
```

Comment:
First one uses `/` between url parts and the second one doesn't, is it by design ?